perses/rjmc/topology_proposal.py

```diff
@@ -1604,7 +1604,7 @@ def enumerate_cycle_basis(molecule):
                 for i in range(len(cycle)):
                     atom_index_1 = cycle[i]
                     atom_index_2 = cycle[(i+1)%len(cycle)]
-                    edge = g.edges[atom_index_1,atom_index_2]
+                    edge = g.edge[atom_index_1][atom_index_2]
```

Comment:
@brycestx : Are you using `networkx` 1.x or 2.x? I thought the API changed for 2.0, and that's what I was using here. Or are both `Graph.edge` and `Graph.edges` available in 2.0?

Reply:
It does appear that my networkx is 1.11. I just upgraded and it works now. My bad, we should add a requirement in setup.py to address this

Reply:
We've had some trouble with requirements in `setup.py` that can't be installed via `pip`---I wonder if there's a way to just add a function to `setup.py` that checks to make sure all the requirements in `devtools/conda/meta.yaml` are installed and suggest you install them via `conda` if they aren't present.